src/lib/builders/toolbar/create.ts

```diff
@@ -137,7 +137,8 @@ export const createToolbar = (props?: CreateToolbarProps) => {
 						'data-disabled': disabledAttr(disabled),
 						'data-value': itemValue,
 						'data-state': pressed ? 'on' : 'off',
-						'aria-pressed': pressed,
+						'aria-checked': $type === 'multiple' ? undefined : pressed,
+						'aria-pressed': $type === 'single' ? undefined : pressed,
```

Comment:
This logic is a bit confusing, as its inverted. Maybe we should do the following
```suggestion
						'aria-checked': $type === 'single' ? pressed : undefined,
						'aria-pressed': $type === 'multiple' ? pressed : undefined,
```